Invalid rule: [[fe80:53:*]] - wildcards can only be used as a suffix

<!-- Thanks for the bug report! -->

## *Who* is the bug affecting?

dnscrypt-proxy 2.0.42

## *What* is affected by this bug?

Error message shows up at the time dnscrypt-proxy start up,

```
[ERROR] Invalid rule: [[fe80:53:*]] - wildcards can only be used as a suffix at line 12
```

With `ip-blacklist.txt` config file has configured as follow, (copied from `example-ip-blacklist.txt`)

```
163.5.1.4
94.46.118.*
[fe80:53:*]        # IPv6 prefix example
```

## *When* does this occur?

The dnscrypt-proxy start up.

## *How* do we replicate the issue?
<!-- Please list all the steps required to reliably replicate it, starting from a newly installed operating system -->

1. Enable `ip-blacklist` feature
2. Start up dnscrypt-proxy

--- a/dnscrypt-proxy/example-ip-blacklist.txt
+++ b/dnscrypt-proxy/example-ip-blacklist.txt
@@ -10,4 +10,4 @@
 
 163.5.1.4
 94.46.118.*
-[fe80:53:*]        # IPv6 prefix example
+fe80:53:*          # IPv6 prefix example

